extensions/wikia/FacebookClient/scripts/FacebookClient.XFBML.js

```diff
@@ -2,19 +2,34 @@
 
 /**
  * Render FB XML tags emebedded into wikitext
- * This file is loaded via ResourceLoader only when FB XML tags are present on the page
  */
 $(function () {
 	'use strict';
 
 	// load api on page load and on demand
 	mw.hook('wikipage.content').add(function ($content) {
-		$.loadFacebookAPI()
-			.done(function () {
-				$('.sso-login').removeClass('hidden');
 
-				// scan the new content for any fb tags
-				FB.XFBML.parse($content[0]);
-			});
+		if ((xfbmlTagsOnPage())) {
+			if (facebookSDKNotLoaded()) {
+				// XFBML tags rendered automatically when SDK loads
+				$.loadFacebookSDK();
+			} else {
+				renderXFBMLTags();
```

Comment:
@jsutterfield, sorry about the trouble you ran into with `$.loadFacebookSDK`. I actually noticed that bug and fixed it in [this PR](https://github.com/Wikia/app/pull/6296/files#diff-d8d8cc9a4a23644a051db21dc3b2ce3aL49). Once that's merged you shouldn't need to do these work-arounds, you should just be able to call `$.loadFacebookSDK().done(...)` and it should work fine after that. Again, sorry for the bug and for not communicating that. It looks like it would have saved you some time.